src/ddmd/aggregate.h

```diff
@@ -275,11 +275,11 @@ class ClassDeclaration : public AggregateDeclaration
     TypeInfoClassDeclaration *vclassinfo;       // the ClassInfo object for this ClassDeclaration
     bool com;                           // true if this is a COM class (meaning it derives from IUnknown)
     bool cpp;                           // true if this is a C++ interface
+    bool isObjc;                        // true if this is an Objective-C class/interface
```

Comment:
Another name nit. Note the subsequent `isscope` and `isabstract` as all lower case. `isObjc` looks like a function name, perhaps `isobjc` is better and more consistent with other usage.